prowler/providers/iac/iac_provider.py

```diff
@@ -308,37 +361,39 @@ def run_scan(
 
             reports = []
 
-            # If only one framework has findings, the output is a dict, otherwise it's a list of dicts
-            if isinstance(output, dict):
-                output = [output]
-
-            # Process all frameworks findings
+            # Process all trivy findings
             for finding in output:
-                results = finding.get("results", {})
 
-                # Process failed checks
-                failed_checks = results.get("failed_checks", [])
-                for failed_check in failed_checks:
-                    report = self._process_check(finding, failed_check, "FAIL")
+                # Process Misconfigurations
+                for misconfiguration in finding.get("Misconfigurations", []):
+                    report = self._process_finding(
+                        misconfiguration, finding["Target"], finding["Type"]
+                    )
                     reports.append(report)
-
-                # Process passed checks
-                passed_checks = results.get("passed_checks", [])
-                for passed_check in passed_checks:
-                    report = self._process_check(finding, passed_check, "PASS")
+                # Process Vulnerabilities
+                for vulnerability in finding.get("Vulnerabilities", []):
+                    report = self._process_finding(
+                        vulnerability, finding["Target"], finding["Type"]
+                    )
                     reports.append(report)
-
-                # Process skipped checks (muted)
-                skipped_checks = results.get("skipped_checks", [])
-                for skipped_check in skipped_checks:
-                    report = self._process_check(finding, skipped_check, "MUTED")
+                # Process Secrets
+                for secret in finding.get("Secrets", []):
+                    report = self._process_finding(
+                        secret, finding["Target"], finding["Class"]
+                    )
+                    reports.append(report)
+                # Process Licenses
+                for license in finding.get("Licenses", []):
+                    report = self._process_finding(
+                        license, finding["Target"], finding["Type"]
+                    )
                     reports.append(report)
 
             return reports
 
         except Exception as error:
-            if "No such file or directory: 'checkov'" in str(error):
-                logger.critical("Please, install checkov using 'pip install checkov'")
+            if "No such file or directory: 'trivy'" in str(error):
+                logger.critical("Please, install trivy using 'pip install trivy'")
```

Comment:
Trivy is a go binary. Should we point to their [docs](https://github.com/aquasecurity/trivy#get-trivy)?